### PR TITLE
Returning struct instead of tuple

### PR DESCRIPTION
**Pull request by hackaugusto**
## Describe your changes

A small change for readability sake.

## Checklist before requesting a review
- Repo forked and branch created from `next` according to naming convention.
- Commit messages and codestyle follow [conventions](./CONTRIBUTING.md).
- Relevant issues are linked in the PR description.
- Tests added for new functionality.
- Documentation/comments updated according to changes.

### CODE REVIEW

File: stdlib/tests/crypto/stark/verifier_recursive/mod.rs [hunk 1]
```diff
@@ -13,16 +13,23 @@ use channel::VerifierChannel;
 pub const BLOWUP_FACTOR: usize = 8;
 pub type QuadExt = QuadExtension<Felt>;
 
+pub struct VerifierData {
```
Comment (vlopes11) [inline stdlib/tests/crypto/stark/verifier_recursive/mod.rs:16]: nit: Maybe add some convenience proc derive?

```rust
#[derive(Debug, Clone, Eq, PartialEq)]
```

**Review by vlopes11 — COMMENTED** [review]
Looks good! I'd add a bit of documentation to the new `VerifierData` - at least a one-liner describing what is the type and where it is expected to be used

**Review by bobbinth — APPROVED** [review]
Looks good! Thank you!

File: stdlib/tests/crypto/stark/verifier_recursive/mod.rs (hunk 1 above)
Comment (hackaugusto) [reply stdlib/tests/crypto/stark/verifier_recursive/mod.rs:17]: added

**Pull request merged**